### PR TITLE
feat(404): add simple custom 404 page

### PR DESCRIPTION
**Pull request by trieloff**
see #298

try it at https://custom-404--theblog--adobe.hlx.page/nothere

### CODE REVIEW

**Comment by davidnuescheler** [comment]
i think i could fix the layout a little bit 😄

**Comment by rofe** [comment]
I wouldn't make it a (shareable) post though...

File: scripts.js [hunk 1]
```diff
@@ -197,6 +197,9 @@ window.blog = function() {
       }
     }
   }
+  if (window.isErrorPage) {
+    pageType = TYPE.POST;
```
Comment (rofe) [inline scripts.js:201]: Why? I don't think this page has much in common with a post page.

**Comment by davidnuescheler** [comment]
i changed things around and merged this branch into `staging`... feel free to test here:
https://staging--theblog--adobe.hlx.page/nothere

**Comment by rofe** [comment]
> i changed things around and merged this branch into `staging`... feel free to test here:
> https://staging--theblog--adobe.hlx.page/nothere

@davidnuescheler @trieloff I see a homepage?

**Comment by trieloff** [comment]
Merge directly or go through staging?

**Comment by avanishgandhi** [comment]
can we alight it like the A.com 404 page? 
Blog 404 page
image
The content on the 404 page for blog is left alight to the window. The design guidelines are that content should be aligned with where the nav starts on the left side .

A.com - 404 page
image

**Comment by avanishgandhi** [comment]
Also our CTA buttons style and colors need to match the A.com standard CTA styles. We can tackle this as a change for batch 2, but just wanted to try and be consistent.

**Pull request merged**